repeat_test_access.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+role='"yellow"'
+echo $role
+while [ $role == '"yellow"' ]
```

Comment:
all on one line. Look through code for examples of syntax to use.

Reply:
Oh no that's a mistake. I think the script made it in the latest checkin. That's purely diagnostic. reverting.

Reply:
Reverted.